docs/aurora/data-science/frameworks/pytorch.md

```diff
@@ -1 +1,321 @@
 # PyTorch on Aurora
+
+PyTorch is a popular, open source deep learning framework developed and 
+released by Facebook. The [PyTorch home page](https://pytorch.org/), has more
+information about PyTorch, which you can refer to. For troubleshooting on 
+Aurora, please contact support@alcf.anl.gov.
+
+## Installation on Aurora
+
+PyTorch is already installed on Aurora with GPU support and available through
+the frameworks module. To use it from a compute node, please load the following modules:
+
+```
+module use /soft/modulefiles/
+module load frameworks/2023.12.15.001
+```
+Then you can load PyTorch as usual, the following is an output from the
```

Comment:
that does not make sense do you mean they load module pytorch, if so then please clearly indicate that

Reply:
Thanks for pointing this out!
"Then you can `import` PyTorch as usual ..." should clarify the confusion. I have updated the document, please take a look.